NorthstarDedicatedTest/modmanager.cpp

```diff
@@ -333,6 +364,27 @@ void ModManager::LoadMods()
 					if (m_hasLoadedMods && modVpk.m_bAutoLoad)
 						(*g_Filesystem)->m_vtable->MountVPK(*g_Filesystem, vpkName.c_str());
 				}
+				// Load vanilla vpks based on vpk.json
+				if (bUseVPKJson && dVpkJson.HasMember("Preload") && dVpkJson["Preload"].IsObject())
+				{
+					rapidjson::Value::MemberIterator v;
+					auto matchesV = [&v](const ModVPKEntry& match) { return match.m_sVpkPath != v->name.GetString(); };
+
+					for (v = dVpkJson["Preload"].MemberBegin(); v != dVpkJson["Preload"].MemberEnd(); v++)
+					{
+						// if it's a vpk that was taken care of in the previous for loop, skip it.
+						bool shouldSkip = std::find_if(mod.Vpks.begin(), mod.Vpks.end(), matchesV) != mod.Vpks.end();
+						if (shouldSkip)
+							continue;
+						if (!v->value.IsTrue())
+							continue;
+						ModVPKEntry& modVpk = mod.Vpks.emplace_back();
+						modVpk.m_bAutoLoad = true;
+						std::string path = "vpk/client_";
+						path += v->name.GetString(); // for some reason these have to be done separately but fine
+						modVpk.m_sVpkPath = path;
```

Comment:
Why don't you call MountVPK here like we do for mod vpks?

Reply:
Moved to another PR for now